hpx/lcos/local/dataflow.hpp

```diff
@@ -81,13 +81,13 @@ namespace hpx { namespace lcos { namespace local
         template <typename Policy, typename Func, typename Futures>
         struct dataflow_frame //-V690
           : hpx::lcos::detail::future_data<
-                typename hpx::util::invoke_fused_result_of<
+                typename hpx::util::detail::fused_result_of<
```

Comment:
Because `decltype` support is required now, result traits and the TR1 protocol become obsolete.